NuRadioReco/framework/parameters.py

```diff
@@ -58,6 +58,7 @@ class channelParameters(Enum):
     signal_ray_type = 16        #: type of the ray propagation path of the signal received by this channel. Options are direct, reflected and refracted
     signal_receiving_azimuth = 17   #: the azimuth angle of direction at which the radio signal arrived at the antenna
     block_offsets = 18 #: 'block' or pedestal offsets. See `NuRadioReco.modules.RNO_G.channelBlockOffsetFitter`
+    radiant_aux_trigger_threshold = 19  #: the threshold of the radiant aux trigger
```

Comment:
Since the trigger threshold is adjusted for each trace (mimicking the threshold servo in the field) the threshold differs for each channel and each event. I don't really know how to store that. This was one idea, but maybe that's overdoing it and we dont need to save it.